server/sonar-server/src/main/java/org/sonar/server/ws/WsUtils.java

```diff
@@ -52,7 +55,7 @@ public static void writeProtobuf(Message msg, Request request, Response response
         msg.writeTo(output);
       } else {
         response.stream().setMediaType(JSON);
-        try (JsonWriter writer = JsonWriter.of(new CacheWriter(new OutputStreamWriter(output, UTF_8)))) {
+        try (JsonWriter writer = JsonWriter.of(new OutputStreamWriter(output, UTF_8))) {
```

Comment:
streaming is reestablished for memory optimization because I fixed the log management in case there's a bug in the ProtobufJsonFormat. After discussions, I thought the trade-off went on optimization side